tests/unit/layout-specs.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import {expect} from 'chai';
+import {ReactWrapper, mount} from 'enzyme';
+
+import {createMemoryHistory, History} from 'history';
+
+import {Layout} from '../../src/components/Layout';
+import PreLoader from '../../src/components/PreLoader';
+import {MainPage} from "../../src/components/pages/MainPage/MainPage";
+import {ContactPage} from "../../src/components/pages/ContactPage/ContactPage";
+import {SideBar} from "../../src/components/SideBar";
+import {Header} from "../../src/components/Header";
+import {SoundSwitch} from "../../src/components/SoundSwitch";
+import {LayoutProps} from "../../src/components/Layout/LayoutProps";
+const preLoaderElement = document.getElementById('pre-loader');
+
+describe('<Layout>', () => {
```

Comment:
This specification should have tests:
- Should hide preloader on mount
- Should show preloader on unmount